mne/evoked.py

```diff
@@ -379,6 +379,11 @@ def plot_white(self, noise_cov, show=True):
             The noise covariance as computed by ``mne.cov.compute_covariance``.
         show : bool
             Whether to show the figure or not. Defaults to True.
+        rank : dict of int | None
+            Dict of ints where keys are 'eeg', 'mag' or 'grad'. If None,
```

Comment:
it can't be meg?

Reply:
needs update as @agramfort mentioned earlier, can now be `meg` right?